plugins/git/functions

```diff
@@ -107,7 +107,7 @@ git_hook_cmd() {
   local APP="$2"  
   local DOKKU_DEPLOY_BRANCH="$(git_deploy_branch "$APP")"
 
-  if ! git check-ref-format --branch "$DOKKU_DEPLOY_BRANCH" 2> /dev/null; then
+  if ! git check-ref-format --branch "$DOKKU_DEPLOY_BRANCH" > /dev/null 2>&1; then
```

Comment:
This outputs the branch name, though we actually don't care about the output, so long as the command returns zero (meaning its a valid branch name).